Scala/Scala.sublime-syntax

```diff
@@ -990,6 +990,19 @@ contexts:
           pop: true
         - match: \n
           scope: invalid.string.newline.scala
+    - match: '\b(raw)(")'
+      captures:
+        1: support.function.scala
```

Comment:
In Python I decided to use `storage.modifier` for this, but it's currently unclear whether this is the desired scope for string modifiers such as raw strings. I'm unsure where this was discussed, however.